src/include/ipc4/copier.h

```diff
@@ -350,6 +349,7 @@ struct copier_data {
 	pcm_converter_func converter[IPC4_COPIER_MODULE_OUTPUT_PINS_COUNT];
 	uint64_t input_total_data_processed;
 	uint64_t output_total_data_processed;
+	struct ipc4_copier_module_cfg config;
 };
 
```

Comment:
I wonder how did this work until now?

Reply:
Wonder why this passed build ? @lyakh have you been able to build this with GCC ?

Reply:
@dbaluta as @RanderWang explained, it wasn't used. And indeed in some cases that array is large, and if we don't need to keep that configuration, it's ok to not do so, but then we need to make it explicit in the code.
@lgirdwood sure, it builds both ways, because that array isn't declared as variable size, it's declared as size 1 because of problems with some compilers I guess